applications/GeoMechanicsApplication/tests/test_curved_beam_elements.py

```diff
@@ -0,0 +1,42 @@
+import sys
+import os
+
+sys.path.append(os.path.join('..', '..', '..'))
+sys.path.append(os.path.join('..', 'python_scripts'))
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import test_helper
+
+class KratosGeoMechanicsCurvedBeamElementTests(KratosUnittest.TestCase):
+    """
+    This class contains benchmark tests which are checked with the analytical solution
+    """
+
+    def setUp(self):
+        # Code here will be placed BEFORE every test in this TestCase.
+        pass
+
+    def tearDown(self):
+        # Code here will be placed AFTER every test in this TestCase.
+        pass
+
+    def test_inclined_beam_2D3N(self):
+        test_name = 'test_inclined_beam_2D3N'
+        file_path = test_helper.get_file_path(os.path.join('.', test_name + '.gid'))
+        simulation = test_helper.run_kratos(file_path)
+
+        displacements = test_helper.get_displacement(simulation)
+        x_displacements = [displacement[0] for displacement in displacements]
+
+        node_number = 15
+        expected_value = 32.337500
+        self.assertAlmostEqual(expected_value, x_displacements[node_number-1], 6)
+
+
+if __name__ == '__main__':
+    suites = KratosUnittest.KratosSuites
+    smallSuite = suites['small'] # These tests are executed by the continuous integration tool
+    smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([KratosGeoMechanicsCurvedBeamElementTests]))
+    allSuite = suites['all']
+    allSuite.addTests(smallSuite)
+    KratosUnittest.runTests(suites)
```

Comment:
```suggestion
    KratosUnittest.main()
```

this should do the same, check the tests in the structural